managers/templates/managers/project_expected_hours.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'base.html' %}
```

Comment:
Same here with a template name. It should be more precise

Reply:
This refers to the user/templates/base.html template which is used as a base for all other templates. I cannot change it here without introducing changes to all templates. Do we even want to do that?

Reply:
I thought also in previous comment that you didn't understand me:) I couldn't comment name of a file that's why I commented first line. Here, as well as in previous comment [here](https://github.com/Code-Poets/sheetstorm/pull/612#discussion_r654833486) I was talking about name of file which I commented :)

Reply:
renamed to list_project_expected_hours